package.json

```diff
@@ -35,6 +35,7 @@
     "react": "0.12.2",
     "serve-static": "1.9.2",
     "shelljs": "0.4.0",
+    "sinon": ">=1.4.0 <2",
```

Comment:
I don't have a revised opinion now or the bandwidth to think about it, so I'd prefer we just stay consistent. I believe @jh-bate is correct that our policy has been to lock down dependencies everywhere, including devDependencies.

Why is sinon even being added here? I didn't think the tests ran?